qtxdg/xdgdesktopfile.cpp

```diff
@@ -836,9 +836,11 @@ bool XdgDesktopFile::startDetached(const QStringList& urls) const
     {
     case ApplicationType:
         return d->startApplicationDetached(this, urls);
+        break;
```

Comment:
t@jubalh Sure. But the return might be replaced by something else and adding the `break` might be forgot.
Also, it confuses analyzers.